### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by Jim876633** [Jim876633/Jim-Docusaurus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jim876633** [comment]
giscus comment testing 😆

**Comment by Egcoo** [comment]
学习JavaScript的时候分不清 e.target 和e.currentTarget，在google搜索就看到了您之前在另一个网站写的帖子，感谢！

**Reply by Jim876633** [reply]
很高興對你有幫助！感謝回饋～
